apps/examples/testcase/ta_tc/ble_manager/utc/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_BLE_MANAGER_UTC
+	bool "BLE Manager UTC TestCase Example"
+	select BLE_MANAGER
+	default n
+	---help---
+		Enable the BLE Manager TestCase Example
```

Comment:
Do we need a newline character at the end here?

Reply:
Done